makejdk.sh

```diff
@@ -24,17 +24,13 @@
 
 # You can set the JDK boot directory with the JDK_BOOT_DIR environment variable
 
-REPOSITORY=AdoptOpenJDK/openjdk-jdk8u
-OPENJDK_REPO_NAME=openjdk
-
-OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
-OS_MACHINE_NAME=$(uname -m)
 
-JVM_VARIANT=${JVM_VARIANT:-server}
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=sbin/functions.sh
+source "$SCRIPT_DIR/sbin/functions.sh"
 
-BUILD_TYPE=normal
-DEFAULT_BUILD_FULL_NAME=${OS_KERNEL_NAME}-${OS_MACHINE_NAME}-${BUILD_TYPE}-${JVM_VARIANT}-release
-BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
+REPOSITORY=AdoptOpenJDK/openjdk-jdk8u
```

Comment:
Clearly `jdk8u` should be generified*, as we will have other variants to come

*not even a word